lib/hydra/works/models/concerns/generic_work_behavior.rb

```diff
@@ -0,0 +1,60 @@
+module Hydra::Works
+  module GenericWorkBehavior
+    extend ActiveSupport::Concern
```

Comment:
This module should `include Hydra::PCDM::ObjectBehavior` because it references `members` which is provided in that module.